.github/workflows/helm.yaml

```diff
@@ -38,13 +38,10 @@ jobs:
         fetch-depth: 0
 
     - name: Update helm index
-      env:
-        VERSION: ${{ github.event.release.tag_name }}
-        DOWNLOAD_URL: ${{ join(github.event.release.assets.*.browser_download_url, ' ') }}
       run: |
         git config user.name "Github Actions"
         git config user.email "no-reply@github.com"
-        ./hack/update-helm-index.sh --helm-repo-path $HELM_REPO_PATH $DOWNLOAD_URL
+        ./hack/update-helm-index.sh --helm-repo-path $HELM_REPO_PATH --version ${{ github.event.release.tag_name }}
```

Comment:
https://github.com/NVIDIA/k8s-device-plugin/blob/main/hack/update-helm-index.sh#L75

Reply:
Sorry, I don't know what you're saying here?

Reply:
edited